hack/dind-cluster.sh

```diff
@@ -324,11 +324,12 @@ os::provision::install-cmds ${DEPLOYED_ROOT}"
 }
 
 function nodes-are-ready() {
-    local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
+  # Skip the SDN node since nothing is intended to be scheduled on it.
+  local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
 KUBECONFIG=${DEPLOYED_CONFIG_ROOT}/openshift.local.config/master/admin.kubeconfig \
-oc get nodes | grep Ready | wc -l")
-    node_count=$(echo "${node_count}" | tr -d '\r')
-    test "${node_count}" -ge "${NODE_COUNT}"
+oc get nodes | grep -v ${SDN_NODE_NAME} | grep -v NotReady | grep Ready | wc -l")
```

Comment:
Maybe `grep -v SchedulingDisabled` instead of `grep -v ${SDN_NODE_NAME}`?

(Also, we really shouldn't be calling the master's node "the SDN node"... that suggests it's somehow important to the overall functioning of the SDN, which it isn't.)